src/ui/buttons/buttonDesigns/divideLogoAnimation.jsx

```diff
@@ -0,0 +1,129 @@
+/* @flow */
+/** @jsx node */
+import { LOGO_CLASS } from '@paypal/sdk-logos/src';
+import { type ElementNode } from 'jsx-pragmatic/src';
+
+import { CLASS } from '../../../constants';
+import { BUTTON_SIZE_STYLE } from '../config';
+
+type DivideLogoAnimationProps = {|
+    paypalLabelContainerElement : ElementNode,
+    paypalLogoStartingLeftPosition : string
+|};
+
+
+const DESIGN_CONFIG = {
+    min:                            BUTTON_SIZE_STYLE.large.minWidth,
+    max:                            BUTTON_SIZE_STYLE.huge.maxWidth,
+    cssUtilClasses: {
+        DOM_READY:                  CLASS.DOM_READY,
+        PAYPAL_LOGO:                LOGO_CLASS.LOGO,
+        PAYPAL_BUTTON_LABEL:        CLASS.BUTTON_LABEL
+    }
+};
+
+// Returns props necessary to render the animation as long as they are valid
+const getValidDesignProps = function(document, configuration) : DivideLogoAnimationProps | null {
```

Comment:
Maybe? Certain parts can be shared but not all of them. Since it is passed in as a string I'm not sure if it would be better or worse...

Reply:
@jrod1032 it can be generalized, but I am afraid for some animations these properties are going to change, so creating a util for this is not very useful ( in my humble opinion )